tests/test_class_attributes.rs

```diff
@@ -96,11 +96,10 @@ fn recursive_class_attributes() {
 }
 
 #[test]
-#[should_panic(
-    expected = "An error occurred while initializing `BrokenClass.fails_to_init`: \
-                ValueError: failed to create class attribute"
-)]
+#[cfg(panic = "unwind")]
```

Comment:
I think this config flag was stabilized only in Rust 1.60.0: https://releases.rs/docs/1.60.0/#language

Reply:
But maybe we can just move `CaptureStdError` into the test definition itself to avoid the unused warnings?

Reply:
Yes agreed, I'll do when I have a chance in the next day or two ...

Reply:
Added a single fix-up and let bors do its thing.

Reply:
Many thanks!